fdbclient/ServerKnobs.cpp

```diff
@@ -814,6 +815,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( STORAGE_COMMIT_INTERVAL,                               0.5 ); if( randomize && BUGGIFY ) STORAGE_COMMIT_INTERVAL = 2.0;
 	init( BYTE_SAMPLING_FACTOR,                                  250 ); //cannot buggify because of differences in restarting tests
 	init( BYTE_SAMPLING_OVERHEAD,                                100 );
+	init( MIN_BYTE_SAMPLING_PROBABILITY,                           0 ); //Adjustable only for test of PhysicalShardMove. should be always zero for other cases
```

Comment:
nit: // Adjustable only for test of PhysicalShardMove. should be always zero for other cases